config/transformer/models_list

```diff
@@ -5,3 +5,5 @@ openconfig-acl-annot.yang
 openconfig-network-instance.yang
 openconfig-network-instance-annot.yang
 sonic-vxlan.yang
```

Comment:
This is not part of your code. But, can you please remove the sonic-vxlan.yang for others? The models_list is intended for only openconfig modules. thanks

Reply:
Done